cardano-cli/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog for cardano-cli
 
+## 9.3.0.0
+
+- Add `query spo-stake-distribution` command to obtain stake distribution for SPOs
+  (feature)
+  [PR 854](https://github.com/IntersectMBO/cardano-cli/pull/854)
```

Comment:
whats the difference to the currently available query?

```
${cardanocli} ${cliEra} query stake-distribution
                           PoolId                                 Stake frac
------------------------------------------------------------------------------
pool1qfjpgpy83v9hal8z3hjnt3namnc8xcrqstgp504sr2p2krfu7dd   3.330e-4
pool1zm3vayxu0zkz830mqf7p8jhmpngrud292uqyt835cvrlcgxyd8q   5.284e-10
pool1rdaxrw3722f0x3nx4uam9u9c6dh9qqd2g83r2uyllf53qmmj5uu   3.060e-4
pool1yzfhssfm83zlpxvts4degrf6a3xntcgq32adpydd0nrpcme93dn   2.981e-6
pool1y36w7zpgywq9uwkv444cksnftenvynernzq70mhp4808gvj7ahk   3.629e-4
pool1xrr9pc5wrh6p30mgsx5f87lcwgzrk53dylt6mr4dgs7gyy7vrl8   3.238e-4
pool1838mjns693tyn2rs4mj6wrepe4jgqlracwrr9m7270vjzjy3m0x   3.147e-4
pool1t8h2l7t8qrldk3jtk8ermr72ahfaxctx2urvju8kspu2zdt64xf   3.090e-4
pool1tgeeesz363v6emrw5ke37p6v9vtdlm5pr0cakhptula5cy322xn   3.091e-4
pool1sphaf7jkd0495f70zhyqjhcv2zzduwft59g7hhkr5ec967pswdg   3.026e-4
pool13lr6wv9m55el9ahscn8qmutc80gq9m5ajgawjstj3v5rqm7fpd9   2.996e-4
pool1j095q9dhnc9snmpstxqpsvzjj9sp36tyehlmefe4lzkww0cxhw9   3.120e-4
pool1nsc08xgfv4dyqt8wq56r4vqvzj874t84ywfcdefwyry9vpqpclm   3.026e-4
pool15862wp3pxp40hqer5whrkvrwqxj4hwem90g0en0rrz4sysnvd8j   3.076e-4
pool15d8cmfkre298gyujlfkameken3ctv23kp76hevdkn9jfwr0sxhj   3.089e-4
pool1ky4eswx50ka8zl0ntlwyhagu2ply6xdseur77clffa92vu9ygte   3.024e-4
pool1h722gdd6kx782me0v9zresnds4ajugneq0vl9a0ju7mgvemezdf   2.996e-4
pool1eyp42ul6fqx82eq2u8wjg3dqz62h72k7gsajrl82cgk5wjzd7rc   3.088e-4
pool1evnszgj6cnhjuthqk763qlsp2syy6pmupdkgxkuray047h48fxw   3.100e-4
pool1em0u48dlkqr0nuzac06wtskq8flku5mtu64vh2ueq39mq5z92d4   3.034e-4
pool1mx6kzngl36el3y7z6tk5q3xnqsreapjy6ja7c9yn6dp3ygytkzp   3.090e-4
pool1m840r7thpj9mxhw33hud3g9jkvjvm00wkw89ul8t06nngjewl6m   4.428e-4
pool1mmpem42nrq9l0zeqvfv2hguxz5mmrp2cqzrv50mzw9hc6d6pa9h   3.225e-4
pool1ut73ekymhf2ac3t4wjklggpyps52907m84rqy70y20zh5e33ken   3.089e-4
pool1amrnegn99cdwj2zjr90lx2yamxg6s0d7t7vf7phpe94kga9gg86   3.130e-4
pool17wrwuxfv757zdkn527ma3xf2urjj5p0urjyvt4jn4yqzzds3cjj   3.090e-4
pool17h7a64rflza7mdexyf7l93h7464w4au65kt6r7mljud0gr4pp6j   3.112e-4
```

Reply:
ah, get it, the new query will return the absolute delegated stake to a pool in lovelaces right? not a fraction amount.